test/integration/apiserver/flowcontrol/fight_test.go

```diff
@@ -119,25 +117,22 @@ func (ft *fightTest) countWrite(fs *flowcontrol.FlowSchema) {
 }
 
 func (ft *fightTest) createController(invert bool, i int) {
-	fieldMgr := fmt.Sprintf("testController%d%v", i, invert)
+	fieldMgr := fmt.Sprintf("testController-inverting-is-%v", invert)
```

Comment:
Thanks for having a go at simplifiying this based on https://github.com/kubernetes/kubernetes/pull/99935#discussion_r590385960.

Looking at the current code, could we still reduce to a single name field that is also used as the field manager by including both the invert status and the index `i` in the name as you did one line down?  I think that gives you a prefix to search the logs by and an intent of `invert` written the log and the fieldmanager fields.

Reply:
The scenario this test aims to run is the midst of a roll out of a new release into a set of apiservers.  In this scenario there are just two values used for FieldManager: the one from the old release, and the one from the new release.

As I noted in the discussion with @wojtek-t : this is an integration test, and I prefer to not make really precise assumptions about how the servers treat the FieldManager values.  That is why I am aiming for high fidelity here with the actual scenario of concern.